index.html

```diff
@@ -453,26 +557,63 @@
           <dd>
             -
           </dd>
+          <dt>
+            DOMString format = null
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            DOMString unit = null
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            double near = null
```

Comment:
The Web Platform APIs use `double` even if `short` would be more appropriate. Probably due to the fact that `double` maps so nicely to ECMAScript `Number`:

http://heycam.github.io/webidl/#es-double

I suggest we stick with `double`.